forgerock-openbanking-uk-aspsp-common/src/main/java/com/forgerock/openbanking/common/model/data/FRCustomerInfo.java

```diff
@@ -65,6 +66,6 @@ public FRCustomerInfo(CustomerInfo customerInfo){
     @DateTimeFormat(
             iso = DateTimeFormat.ISO.DATE
     )
-    private LocalDate birthdate;
+    private DateTime birthdate;
     private FRCustomerInfoAddress address;
```

Comment:
Changed birthdate from LocalDate to DateTime to avoid array representation when mapping to json to present the value on the UI.